lib/mapping/CMap.cpp

```diff
@@ -719,7 +707,7 @@ CMapEditManager * CMap::getEditManager()
 	return editManager.get();
 }
 
-void CMap::resetStaticData()
+void CMap::resetStaticData() const

```

Comment:
I think methods like this should not have `const` added.
Yes, technically they do quality for const since method makes no changes to object state.
But logically, such methods should not be marked as const - since they should not be used with const object.

In other words, if method has some (signigicant) side effects, it should not be marked const.

Reply:
removed